extensions/binding/org.eclipse.smarthome.binding.meteoblue/README.md

```diff
@@ -0,0 +1,161 @@
+| longitude                | Number             | Geographic longitude of the location (in degrees) |
+| height                   | Number:Length      | Altitude above sea-level of the location (in meters) |
+| forecastDate             | DateTime           | Forecast date |
+| UVIndex                  | Number             | UltraViolet radiation index at ground level (0-16) |
+| minTemperature           | Number:Temperature | Low temperature |
+| maxTemperature           | Number:Temperature | High temperature |
+| meanTemperature          | Number:Temperature | Mean temperature |
+| feltTemperatureMin       | Number:Temperature | Low "feels like" temperature |
+| feltTemperatureMax       | Number:Temperature | High "feels like" temperature |
+| relativeHumidityMin      | Number             | Low relative humidity |
+| relativeHumidityMax      | Number             | High relative humidity |
+| relativeHumidityMean     | Number             | Mean relative humidity |
+| precipitationProbability | Number             | Percentage probability of precipitation |
+| precipitation            | Number:Length      | Total precipitation (water amount) |
+| convectivePrecipitation  | Number:Length      | Total rainfall (water amount) |
+| rainSpot                 | String             | Precipitation distribution around the location |
+| rainArea                 | Image              | Color-coded image generated from rainSpot |
+| snowFraction             | Number             | Percentage of precipitation falling as snow |
+| snowFall                 | Number:Length      | Total snowfall (calculated) |
+| cardinalWindDirection    | String             | Name of the wind direction (eg. N, S, E, W, etc.) |
+| windDirection            | Number             | Wind direction (in degrees) |
+| minWindSpeed             | Number:Speed       | Low wind speed  |
+| maxWindSpeed             | Number:Speed       | High wind speed |
+| meanWindSpeed            | Number:Speed       | Mean wind speed |
+| minSeaLevelPressure      | Number:Pressure    | Low sea level pressure  |
+| maxSeaLevelPressure      | Number:Pressure    | High sea level pressure |
+| meanSeaLevelPressure     | Number:Pressure    | Mean sea level pressure |
+| pictocode                | Number             | Number indicating the weather icon for representing the forecast |
+| icon                     | Image              | The image used for the pictocode (calculated) |
+| predictability           | Number             | Estimated certainty of the forecast (percentage) |
+| predictabilityClass      | Number             | Range 0-5 (0=very low, 5=very high) |
+| precipitationHours       | Number             | Total hours of the day with precipitation |
+| humidityGreater90Hours   | Number             | Total hours of the day with relative humidity greater than 90% |
+
+The following channels are not specific to any day's forecast.
```

Comment:
So `locationName` has already been removed.
`latitude`, `longitude`, `height` and the others, I'm going to argue for keeping because they could be useful to have.

Reply:
Regarding these 3 channels: latitude, longitude, height. **All** of them are included in location, so whats the point for introducing redundancy here?

Reply:
`location` is only used in configuration.
If someone wants to display the latitude and longitude, they have to exist as channels, right? If there's a better way, fine, but I'm not seeing it...

Reply:
As kai already mentioned. You put the information about lat/lon into the thing configuration and this input would be the output of you channels. This doesn't make much sense, i.e. it doesn't create any value. If a user wants to display the coordinates somewhere he can easily take those he entered into the configuration.

Channels in a Thing are supposed to describe functionalities of a Thing. Printing out a configuration value is not a functionality of the Thing.

Reply:
This still seems to be open.

Reply:
I've removed the following channels: latitude, longitude, timezoneAbbreviation, utcTimeOffset, modelRunUTC, and modelRunUpdateTimeUTC.